workshop/content/advanced/resource-types/prerequisites/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Prerequisites'
+date: 2021-11-16T20:50:53Z
+weight: 320
+---
+
+### Resource Type Development Tools
+
+Before proceeding to the next section, choose to install the following prerequisites on your machine:
+
+* follow instructions for your operating system to install the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html): as part of this process, follow also notes to install Docker (unless you have already installed it), that is needed for running contract tests for your resource type;
+* follow [steps](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/what-is-cloudformation-cli.html#resource-type-setup) to install the [CloudFormation Command Line Interface (CLI)](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/what-is-cloudformation-cli.html): this includes installing/checking the version of Python installed on your machine, the installation of the CloudFormation CLI and language plugins. You will use the CloudFormation Command Line Interface for operations that include generating a new project for your resource type with source code stubs, validating your resource type specification against the [resource type schema](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-schema.html), running contract tests and submitting your resource type to the CloudFormation registry;
```

Comment:
Can we clarify that first we install version 1.0 and upgrade to version 2.0 once inside the project? The documentation doesn't go into much detail here.

Reply:
The example in the documentation is for the 1.0 to 2.0 upgrade if version 1.0 is installed. Will add an example and will reference the documentation reference - thank you!